defaults/main.yml

```
@@ -36,3 +47,6 @@ certbot_keep_updated: true
 
 # Where to put Certbot when installing from source.
 certbot_dir: /opt/certbot
+
+# Restart this service after successfull certificate creation:
+certbot_deployhook: "service {{certbot_create_standalone_stop_services }} restart"
```

This doesn't work with multiple services (or even one service where it prints `service [u'nginx'] restart` instead.
Try this instead:
```suggestion
certbot_deployhook: "service {{ certbot_create_standalone_stop_services | join(' restart && service') }} restart"
```